plugins/scenario/worker/Worker.ts

```diff
@@ -60,10 +66,14 @@ export async function run<T, U, R>({ bases, config, worker }: WorkerData) {
   }
 
   for (const base of bases) {
-    const world = new World(hreForBase(base), base), dm = world.deploymentManager;
-    const delta = await dm.runDeployScript({ allMissing: true });
-    console.log(`[${base.name}] Deployed ${dm.counter} contracts, spent ${dm.spent} to initialize world 🗺`);
-    console.log(`[${base.name}]\n${dm.diffDelta(delta)}`);
+    const world = new World(base);
+    const dm = world.deploymentManager;
+    await runDeployScript(dm, base.name);
+
+    if (world.auxiliaryDeploymentManager) {
+      await runDeployScript(world.auxiliaryDeploymentManager, base.auxiliaryBase, true);
+    }
```

Comment:
I wonder if we really need this one

Reply:
yep, you're right. it's unnecessary. removed